Library/OcBootManagementLib/BootEntryManagement.c

```diff
@@ -2380,6 +2381,10 @@ OcLoadBootEntry (
              &DmgLoadContext
              );
   if (!EFI_ERROR (Status)) {
+    if (BootEntry->IsAppleInstaller) {
```

Comment:
Could we provide a more descriptive comment here with reasoning?

Reply:
Will add. (Obviously the reasoning per se is scattered around the docs and comments in the PR (!) - but I will add a specific brief comment here.)

Reply:
Added.